cmd/stateless/main.go

```diff
@@ -0,0 +1,234 @@
+package main
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/console/prompt"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/state"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/eth/ethconfig"
+	"github.com/ethereum/go-ethereum/internal/debug"
+	"github.com/ethereum/go-ethereum/internal/flags"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/trie"
+	"github.com/urfave/cli/v2"
+	"go.uber.org/automaxprocs/maxprocs"
+	"os"
+)
+
+var (
+	BlockWitnessFlag = &cli.StringFlag{
+		Name:  "block-witness",
+		Usage: "foo bar",
+	}
+
+	BlockWitness1Flag = &cli.StringFlag{
+		Name:  "witness1",
+		Usage: "foo bar",
```

Comment:
Please fix these up. For all of these: 
- Remove the ones that aren't needed / wanted
- Add proper usage-descriptions
- Use proper names

It doesn't take a lot of time to do, and it makes the work of reviewing a lot easier if there't not a bunch of foos and bars all around